views/pages/fixture_editor.js

```diff
@@ -784,147 +888,193 @@ function getSubmitDialogString() {
+
+  html += getAdditionalAttributes(options.attributes);
+  html += ` v-model="${modelKey}" />`;
   return html;
 }
 
 /**
- * @param {!string} key The key in Vue's data model that this textarea element is bound to.
- * @param {*} property The JS Schema property that this textarea element has to obey.
- * @param {?string} hint A hint to the user that helps filling the right value into this textarea element.
+ * @param {!string} modelKey The Vue reference used in v-model.
+ * @param {!InputOptions} options Additional optional parameters.
  * @returns {!string} The textarea element HTML.
  */
-function textareaInput(key, property, hint) {
+function textareaInput(modelKey, options = {}) {
   let html = `<textarea`;
-  html += getRequiredAttr(property);
-  html += getPlaceholderAttr(hint);
-  html += ` v-model="${key}"></textarea>`;
+  html += getRequiredAttr(options.required);
+  html += getPlaceholderAttr(options.hint);
+  html += getPatternAttr(options.property);
+  html += getMinlengthAttr(options.property);
+  html += getMaxlengthAttr(options.property);
+
+  html += getAdditionalAttributes(options.attributes);
+  html += ` v-model="${modelKey}"></textarea>`;
   return html;
 }
 
 /**
- * @param {!string} key The key in Vue's data model that this number input element is bound to.
- * @param {*} property The JS Schema property that this input element has to obey.
- * @param {?string} hint A hint to the user that helps filling the right value into this input element.
- * @param {!string} [additionalAttributes=''] Additional HTML attributes that shall be added to the input element.
+ * @param {!string} modelKey The Vue reference used in v-model.
+ * @param {!InputOptions} options Additional optional parameters.
  * @returns {!string} The input element HTML.
  */
-function numberInput(key, property, hint, additionalAttributes = ``) {
+function numberInput(modelKey, options = {}) {
   let html = `<input type="number"`;
-  html += getRequiredAttr(property);
+  html += getRequiredAttr(options.required);
+  html += getMinMaxAttributes(modelKey, options);
+  html += ` step="${options.property.type === `integer` ? `1` : `any`}"`;
 
-  if (property.minimum !== undefined) {
-    if (/max/i.test(key)) {
-      const minKey = key.replace(/([mM])ax/, `$1in`);
-      html += ` :min="typeof ${minKey} === 'number' ? ${minKey} : ${property.minimum}"`;
-    }
-    else {
-      html += ` min="${property.minimum}"`;
+  html += getPlaceholderAttr(options.hint);
+  html += getAdditionalAttributes(options.attributes);
+  html += ` v-model.number="${modelKey}" />`;
+
+  return html;
+}
+
+/**
+ * @param {!string} modelKey The Vue reference used in v-model.
+ * @param {!InputOptions} options Additional optional parameters.
+ * @returns {!string} The HTML attribute(s) declaring a minimum and/or maximum value, or an empty string.
+ */
+function getMinMaxAttributes(modelKey, options) {
+  let html = ``;
+
+  if (options.property) {
```

Comment:
Indentation is not correct. And, as I commented earlier, I'd prefer `if (!options.property) return '';` as the first statement in this function to keep the indentation level smaller.

Reply:
I used VSCode's feature to only add some changes to the commit, and it omitted the indentation ...